src/Common/Storage/Commands.Storage/File/Cmdlet/GetAzureStorageFileContent.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet
     using LocalDirectory = System.IO.Directory;
     using LocalPath = System.IO.Path;
 
-    [Cmdlet(VerbsCommon.Get, LocalConstants.FileContentCmdletName, SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = LocalConstants.ShareNameParameterSetName)]
```

Comment:
@EmmaZhu Yes, this is one of the cmdlets on the list that will require a small amount of refactoring - the cmdlet should implement ShouldProcess, and should prompt with a ShouldContinue prompt when attempting to overwrite a file (unless the user specifies -Force, so the cmdlet also needs a Force parameter)